src/components/IngredientFeed.vue

```diff
@@ -1,14 +1,108 @@
 <template>
   <div class="container">
-    <header class="jumbotron">
-      <h3>This is the ingredients page</h3>
-    </header>
-
     <!-- Example of how to use MDB rows to open the centered modal view -->
     <!-- <MDBRow center class="text-center">
       <h3>Vue Teleport Example</h3>
       <ModalView />
     </MDBRow> -->
+    <div class="tool-bar">
+      <div class="tool-bar-container">
+        <button type="button" class="btn">Sort By</button>
+      </div>
+      <div class="tool-bar-container">
+        <div class="form-outline">
+          <input
+            type="search"
+            id="form1"
+            class="form-control"
+            placeholder="Search Ingredients"
+            aria-label="Search"
+          />
+        </div>
+        <!-- TODO replace with an actual material icon -->
+        <button type="button" class="btn btn-primary btn-floating">
+          <i class="fas fa-plus"></i>
+        </button>
+      </div>
+    </div>
+    <!-- TODO see if this can be done in a list -->
+    <!-- TODO it automatically moves to a new row, 
```

Comment:
I think we should set up a button like netflix or instacart; we only display as many items will fit in one row, and then load the others when the next or previous arrow button is clicked.

![image](https://user-images.githubusercontent.com/20624069/201531865-7581fa89-e329-4dd2-9154-1b80153d9f51.png)

![image](https://user-images.githubusercontent.com/20624069/201531851-ef931b21-51a6-4f33-a0af-1e9076ff58a5.png)

![image](https://user-images.githubusercontent.com/20624069/201531882-13d67082-8f83-4650-9f84-b856f3c6ad98.png)

Reply:
its now scrollable but it will need to be a scroll button later (idk how to make it lazy load the next set but ill look into it)